elastic/kibana
# [Security Solution][Entity Analytics] Adding Asset Criticality column to All Hosts tab in Explore pages

This PR adds a new column for Asset Criticality to the "All hosts" tab in the Explore/Hosts page.
If any of the hosts has criticality data assigned, it will be displayed in the new column. If no criticality has been assigned, the field is left blank.



![Screenshot 2024-06-18 at 15 00 46](https://github.com/elastic/kibana/assets/2423976/dc6e76c9-ceed-4da6-905c-b374181fe0f3)


### How to test

1. Make sure you have test data with Asset Criticality.
    - Either use the [datagen tool](https://github.com/elastic/security-documents-generator) or just make sure to assign criticality to already existing hosts
2. Enable Asset Criticality settings in `Stack Management > Kibana > Advanced Settings`
3. Navigate to `Explore > Hosts > All Hosts`
